src/main/resources/com/sonyericsson/hudson/plugins/gerrit/trigger/GerritServer/index.jelly

```diff
@@ -113,35 +113,35 @@
                         <f:textbox name="gerritBuildStartedVerifiedValue"
                                    value="${it.config.gerritBuildStartedVerifiedValue}"
                                    default="${com.sonyericsson.gerrithudsontrigger.config.Config.DEFAULT_GERRIT_BUILD_STARTED_VERIFIED_VALUE}"
```

Comment:
I don't think all of these default values are possible any more.

I'm going to add some inherital sin on you here and ask you if you can clean the default values up a bit, because they have probably never really worked. I doubt any of them are useful and maybe could just be removed since a new server config should be instantiated with default values from the beginning.